client/src/components/CatalogTourCard/CatalogTourCard.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { styled, Stack, CardActions, CardMedia, Button, Typography, CardContent } from '@mui/material';
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import { ReactComponent as CoinsIcon } from '../../svg/CoinsIcon.svg';
+
+const CardContainer = styled(Stack)(({ theme }) => ({
+  backgroundColor: theme.palette.primary.contrastText,
+  borderRadius: 10,
+  boxShadow: '0px 0px 40px rgba(0, 0, 0, 0.05)',
+}));
+
+const CardImage = styled(CardMedia)(({ theme }) => ({
+  width: '100%',
+  height: 240,
+  borderRadius: '10px 10px 0 0',
+
+  [theme.breakpoints.up('tablet')]: {
+    width: 238,
+    height: 'auto',
+    borderRadius: '10px 0 0 10px',
+  },
+}));
+
+const CardTitle = styled(Typography)(({ theme }) => ({
+  color: theme.palette.text.primary,
+  marginTop: 20,
+  marginBottom: 20,
+}));
+
+const TourDescription = styled(Typography)(({ theme }) => ({
```

Comment:
You can add `variant="body1"` here, as you already using styled component
```suggestion
const TourDescription = styled((props) => <Typography variant="body1" {...props} />)(({ theme }) => ({
```

Reply:
Done